### PR TITLE
s/jenkins.groovy/Jenkinsfile/g

### PR DESCRIPTION
**Pull request by jglick**
At @kohsuke’s request. @reviewbybees

### CODE REVIEW

File: multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 @SuppressWarnings({"unchecked", "rawtypes"}) // core’s fault
 public class WorkflowMultiBranchProject extends MultiBranchProject<WorkflowJob,WorkflowRun> {
 
-    static final String SCRIPT = "jenkins.groovy";
+    static final String SCRIPT = "Jenkinsfile";
```
Comment (stephenc) [inline multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: My preference is that the file name should be configurable by the user but **default** to `Jenkinsfile`. The file is acting as a marker file to identify the branches that should be built. It is rather arrogant to assume that everyone will want to use `Jenkinsfile` as the file name.

OTOH adding the file name being configurable at a future date is not prevented... just more I am stating my opinion - influenced by experience with literate - given that we are touching this behaviour here

Comment (Vlatombe) [inline multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: Not sure it is as important as for Literate. The content of this file is really specific to Jenkins-workflow whereas for Literate, the marker file content is descriptive and could be used in another context.

**Comment by tfennelly** [comment]
:bee:

**Comment by Vlatombe** [comment]
:bee:

File: multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java (hunk 1 above)
Comment (jglick) [inline multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: > the file name should be configurable by the user

I agree, but that is a subject for a future feature (perhaps a form control in `WorkflowBranchProjectFactory`). This is about changing the default filename.

**Pull request merged**